Algorithm.CSharp/BasicTemplateFuturesHistoryAlgorithm.cs

```diff
@@ -54,6 +57,17 @@ public override void Initialize()
             }
 
             SetBenchmark(d => 1000000);
+
+            Schedule.On(DateRules.EveryDay(), TimeRules.Every(TimeSpan.FromHours(1)), MakeHistoryCall);
+        }
+
+        private void MakeHistoryCall()
+        {
+            var history = History(10, Resolution.Minute);
+            if (history.IsNullOrEmpty())
```

Comment:
To be more precise could we assert on the data quantity? Would be nice to have an `OnEndOfAlgorithm` which also asserts the final count or something alike as a safe guard if this method was never called